update-motd.d/99-esm

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+ESM_ENABLED="This Ubuntu 12.04 ESM system is actively receiving extended security updates
+from Canonical:
+ * http://www.ubuntu.com/esm"
+
+ESM_DISABLED="This Ubuntu 12.04 LTS system is past its End of Life, and is no longer
+receiving security updates.  To protect the integrity of this system, it’s
+critical that you enable Extended Security Maintenance updates:
+ * http://www.ubuntu.com/esm"
+
+if `apt-cache policy | grep -q 'esm\.ubuntu\.com'`; then
```

Comment:
You can use 
```
if ubuntu-advantage is-esm-enabled; then
    ...
```

now (I added the command to the new script version). Note that no ` is used (you want the return code, not the output of the command)